app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @public_recipes = Recipe.where(public: true).order(created_at: :desc)
+  end
```

Comment:
- [ ] Kindly fix the `n+1` query that happens in that action `PublicRecipes#index` as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Make%20sure%20there%20are%20no%20N%2B1%20queries%20happening.).